src/main/java/com/axis/jenkins/plugins/eiffel/eiffelbroadcaster/Util.java

```diff
@@ -84,6 +89,29 @@ public static String getFullName(Queue.Task t) {
         }
     }
 
+    /**
+     * Returns the URI of a {@link Run}, or one of its subresources.
+     *
+     * @param r the Run to return the URI for
+     * @param pathSuffix additional path components to append
+     * @return the URI asked for, or null if a URI couldn't be resolved
+     */
+    public static URI getRunUri(final Run r, String... pathSuffix) {
+        Jenkins jenkins = Jenkins.get();
+        if (jenkins.getRootUrl() != null) {
+            try {
+                String uri = Functions.joinPath(jenkins.getRootUrl(), r.getUrl());
+                if (pathSuffix.length == 0) {
+                    return new URI(uri);
```

Comment:
No, I think I stole^W was inspired by MQ Notifier. Wasn't aware of this deprecation. It's annoying since there's no suggested replacement. Might as well use the existing deprecated method instead of working around it like this, but I'll kick that can down the road to not hold up this PR.